docs/concepts/source-ip/source-ip.md

```diff
@@ -0,0 +1,251 @@
+---
+---
+
+{% capture overview %}
+Applications running in a Kubernetes cluster find and communicate with each
+other, and the outside world, through the Service abstraction. This document
+explains what happens to the source IP of packets sent to different types
+of Services, and how you can toggle this behavior according to your needs.
+{% endcapture %}
+
+{% capture body %}
+
+### Terminology
+
+Throughout this document, we will use the following terms:
+
+* [NAT](https://en.wikipedia.org/wiki/Network_address_translation): network address translation
+* [Source NAT](/docs/user-guide/services/#ips-and-vips): replacing the source IP on a packet, usually with a node's IP
+* [Destination NAT](/docs/user-guide/services/#ips-and-vips): replacing the destination IP on a packet, usually with a pod IP
+* [VIP](/docs/user-guide/services/#ips-and-vips): a virtual IP, such as the one assigned to every Kubernetes Service
+* [Kube-proxy](/docs/user-guide/services/#virtual-ips-and-service-proxies): a network daemon that orchestrates Service VIP management on every node
```

Comment:
If we make all implementations of kubeproxy the exact same, we'll never need to use its name I guess. Right now I need it at least to say "iptables proxier preserves intra cluster source ip"